app/Jobs/SendNewRecordingNotifications.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Friend;
+use App\Jobs\Job;
+use App\Phone\Exceptions\BlacklistedPhoneNumberException;
+use App\Phone\TwilioClient;
+use App\PhoneNumber;
+use App\Recording;
+use App\User;
+use Illuminate\Log\Writer as Logger;
+
+class SendNewRecordingNotifications extends Job
```

Comment:
I think I was just trying to get a handle on the places that the blacklisted exception could occur. I see you started on and event/listener setup for sending new recording notifications, maybe firing the CallRecordingWasCompleted event and splitting this back into the event listeners would clean it back up, what do you think?